fireapp/lib/data/client/api/rest_client.dart

```diff
@@ -123,4 +124,13 @@ abstract class RestClient {
   Future<List<Shift>> getShifts(
       @Path("userId") int userId
       );
+  @POST("/v2/register-token")
```

Comment:
Just a heads up, I've instructed Chim to change this endpoint to `POST /v2/user/<user-id>/token`, and the unregister one to `DELETE /v2/user/<user-id>/token`